tempesta_fw/str.h

```diff
@@ -92,13 +92,17 @@
  * @skb		- socket buffer containign the string data;
  * @len		- total length of compund or plain string (HTTP message body
  *		  size can be extreme large, so we need 64 bits to handle it);
+ * @eol_len     - the length of string's line endings, if present (as for now,
+ *                it should be 0 if the string has no EOL at all, 1 for LF and
+ *                2 for CRLF);
  * @flags	- 3 most significant bytes for number of chunks of compound
  * 		  string and the least significant byte for flags;
  */
 typedef struct {
 	void		*ptr;
 	struct sk_buff	*skb;
 	unsigned long	len;
+	unsigned char   eol_len;
```

Comment:
Seems that not `long` not `char` does not assumes any intention of use. The point was that if whole the project doesn't use any 'uintXX_t` types there is no good reason to use `uint8_t` or similar in that case. Also, `char` is a plain-C type witch carries 8-bit signed/unsigned value. That's the only what we needed here.